### ISSUE TITLE
€ sign should be prefixed instead of postfixed

### ISSUE THREAD

**Issue by martinbroos** [foretagsplatsen/numbro#137]
The € sign should always be in front of the amount, when writing out euro then it's postfixed.

€10,-
10 euro

Can this be changed for the languages that use euro signs?

**Comment by BenjaminVanRyseghem** [comment]
> The € sign should always be in front of the amount

This is not true, see [this](https://en.wikipedia.org/wiki/Linguistic_issues_concerning_the_euro)

> Can this be changed for the languages that use euro signs?

Yes, it's a per language setting. For french by example the setting is [here](https://github.com/foretagsplatsen/numbro/blob/develop/languages/fr-FR.js#L28)

**Comment by martinbroos** [comment]
Ah i see I was not aware of this. It looks like the setting is wrong for the dutch lang file. It should be prefixed. Can this be changed?

**Comment by BenjaminVanRyseghem** [comment]
sure it can be :smile:

The dutch language file needs to be updated for that
